encrypt_util_test.go

```diff
@@ -33,7 +33,7 @@ func TestEncryptDecryptFile(t *testing.T) {
 	data := "test data"
 	inputFile := "test_encrypt_decrypt_file"
 
-	fd, err := os.OpenFile(inputFile, os.O_CREATE|os.O_WRONLY, os.ModePerm)
+	fd, err := os.OpenFile(inputFile, os.O_CREATE|os.O_WRONLY, 0666)
```

Comment:
`os.Create`?

Reply:
A general comment about replacing all `os.OpenFile(..., os.O_CREATE, 0666)` with `os.Create`:  
It will affect behavior in a potentially dangerous way, as after the change it will truncate the file when it doesn't exist (i.e. delete its contents).
There's a good chance it won't have any negative effect, but I'd still check it on a case-by-case basis.

Reply:
Yes, you are right. But I tried to focus only on tests :) I didn't mention any `os.Create` related lines in production code.

Reply:
I've changed to `os.Create` in test files.